scripts/dashboard-esbuild.ts

```diff
@@ -0,0 +1,358 @@
+// Node.js-based dashboard build using Group Income's Vue esbuild plugin approach
+// Uses Node.js to avoid Deno compatibility issues with @vue/component-compiler
+
+import { sassPlugin } from 'npm:esbuild-sass-plugin@3.3.1'
+import fs from 'node:fs'
+import { mkdir, readFile, readdir, writeFile } from 'node:fs/promises'
+import { basename, dirname, join, relative, resolve } from 'node:path'
+import process from 'node:process'
+import { fileURLToPath } from 'node:url'
+import componentCompiler, { type SFCCompiler, type StyleCompileResult, type DescriptorCompileResult } from 'npm:@vue/component-compiler'
+import esbuild from 'npm:esbuild'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = dirname(__filename)
+
+// Environment variables setup (following Group Income pattern)
+const {
+  CI = '',
+  LIGHTWEIGHT_CLIENT = 'true',
+  NODE_ENV = 'development',
+  EXPOSE_SBP = '',
+  ENABLE_UNSAFE_NULL_CRYPTO = 'false',
+  UNSAFE_TRUST_ALL_MANIFEST_SIGNING_KEYS = 'false'
+} = process.env
+
+// Read package.json for version info
+const packageJSON = JSON.parse(fs.readFileSync(join(__dirname, '../package.json'), 'utf8'))
+const CONTRACTS_VERSION = packageJSON.contractsVersion || '2.0.0'
+const CHEL_VERSION = packageJSON.version
```

Comment:
Likewise, no idea why this line is here, it should also be removed. The dashboard isn't project-specific.